VirtoCommerce.SitemapsModule.Web/module.manifest

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <module>
     <id>VirtoCommerce.Sitemaps</id>
-    <version>1.1.6</version>
+    <version>1.1.7</version>
```

Comment:
Please do not update module version in PR requests it will be be done before release by release manager